src/haddock/libs/libsubprocess.py

```diff
@@ -45,20 +46,45 @@ def __init__(
             self,
             input_file,
             output_file,
-            cns_folder='.',
+            cns_folder,
+            modpath,
+            config_path,
             cns_exec=None,
             ):
         """
         CNS subprocess.
 
-        :param input_file: input CNS script
-        :param output_file: CNS output
-        :cns_folder: absolute execution path
-        :cns_exec: CNS binary including absolute path
+        Parameters
```

Comment:
Shouldn't this also document the toppa_path?

Reply:
Okay, will commit on that.